Content.Server/Traits/TraitSystem.Functions.cs

```diff
@@ -400,3 +406,75 @@ public override void OnPlayerSpawn(EntityUid uid,
         staminaComponent.Cooldown += CooldownModifier;
     }
 }
+
+// FLOOF START
+[UsedImplicitly]
+public sealed partial class TraitModifyMetabolism : TraitFunction
```

Comment:
This should be moved to floofstation namespace.

Reply:
How would I do that?

Reply:
Just move it to Content.Server/Floofstation/.../... .cs, create the file/namespace if it doesn't exist